bin/changelog

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env node
```

Comment:
Dumb question: Does this work? Now that I am thinking of it, this should work since hashbang is just like "attn run it using this" right?

Reply:
I believe it is needed https://docs.npmjs.com/files/package.json#bin

Reply:
Sorry, I was super non-specific. I meant the space between the hashbang and the command, I haven't seen this before. :)